assets/src/blocks/course-progress/index.js

```diff
@@ -72,22 +68,35 @@ export const settings = {
 	},
 
 	/**
-	 * The save function defines the way in which the different attributes should be combined
-	 * into the final markup, which is then serialized by Gutenberg into post_content.
+	 * Block Editor Save
 	 *
-	 * The "save" property must be specified and must be a valid function.
+	 * Does nothing since it's a dynamic block.
 	 *
-	 * @link https://wordpress.org/gutenberg/handbook/block-api/block-edit-save/
-	 * @param   {Object} props Block properties.
-	 * @return  {Function}
-	 * @since   1.0.0
-	 * @version 1.0.0
+	 * @since 1.0.0
+	 * @since [version] Return null, it's a dynamic block.
+	 *
+	 * @return null
 	 */
-	save: function( props ) {
-		return (
-			<div className={ props.className }>
-				[lifterlms_course_progress]
-			</div>
-		);
+	save: function() {
+		return null;
 	},
+	deprecated: [
```

Comment:
oh deprecated is an array -- when reading the docs yesterday I thought it was an object. That's good, that probably means we can have multiple deprecations should we need them...

Reply:
yeah, I thought that too